includes/admin/views/html-notice-uploads-directory-is-unprotected.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Admin View: Notice - Uploads directory is unprotected.
+ *
+ * @package WooCommerce\Admin\Notices
+ * @since   4.2.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+$uploads = wp_get_upload_dir();
+
+?>
+<div id="message" class="updated woocommerce-message">
```

Comment:
This class should be `error` instead of `updated` to make it render as red/an error.

Reply:
@beaulebens on WooCommerce we don't set any notice as `error`, also only updating this class will render in the same color because of `woocommerce-message`, if it get removed will cause problems for the close button.
Should I go ahead and create styling for it? Maybe we should revise other notice classes too, but currently all notices uses the same wrap as this one.

Reply:
It seemed to work fine when I changed it in my browser. I don't want it to be a blocker to us getting this done, but we should take a look at that across the board and try to use the appropriate classes, based on the type of message being shown.

Reply:
> It seemed to work fine when I changed it in my browser
No luck for me on Firefox:

![Screenshot from 2020-05-07 14-34-47](https://user-images.githubusercontent.com/1264099/81326255-f1c1c900-906f-11ea-882c-2977f153e5e3.png)

> but we should take a look at that across the board and try to use the appropriate classes, based on the type of message being shown.

We force the same border color for all notices, I'll open an issue for us to revise it and include other colors.

Reply:
@beaulebens I updated this PR including the `error` class, but it still doesn't work, I created a patch for the notice styling bug in https://github.com/woocommerce/woocommerce/pull/26410